src/api/app/views/webui2/webui/monitor/_workers_table.html.haml

```diff
@@ -1 +1,48 @@
-%h2 Workers
+%h2.nowrap#workers-title
+  Workers
+  %span.hidden#workers-updating (updating...)
+
+.col-12
```

Comment:
It looks a bit weird now, especially on devices with smaller viewports. It should look better if it's on its own line right under the `h2`.

![monitor-review](https://user-images.githubusercontent.com/1102934/58939044-8a700a80-8776-11e9-95bb-497fa8f2207f.png)